bin/s3scan.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+var keepalive = require('agentkeepalive');
+var agent = new keepalive.HttpsAgent({
+  keepAlive: true,
+  maxSockets: Math.ceil(require('os').cpus().length * 16),
+  keepAliveTimeout: 60000
+});
+var s3scan = require('..');
+var argv = require('minimist')(process.argv);
+
+var s3url = argv._[2];
+if (!s3url) {
+  console.error('Usage: s3scan <s3url> [--concurrency=num] [--gunzip]');
+  process.exit(1);
+}
+
+var options = { agent: agent, body: true };
+if (argv.gunzip) options.gunzip = new Boolean(argv.gunzip);
+if (argv.concurrency) options.concurrency = parseInt(argv.concurrency, 10);
+
+s3scan.Scan(s3url, options).pipe(process.stdout);
```

Comment:
No delimiter -- assumption atm is if you're doing something with line-delimited files your files have terminating `\n`'s as well. I think this is a pretty good assumption, would be like doing

```
cat a.csv b.csv c.csv
```

But via s3scan.